OpenMarket/OpenMarket/Extensions/UIViewController+extension.swift

```diff
@@ -7,11 +7,23 @@
 
 import UIKit
 
+private extension OpenMarketEnum {
+    static let close = "닫기"
+}
+
 extension UIViewController {
     func showAlert(alertTitle: String) {
```

Comment:
alert를 두번 중복해서 사용하지 않아도 될 것 같아요.
```suggestion
    func showAlert(title: String) {
```

Reply:
`showAlert(title: String)`으로 변경했습니다.